src/validate.js

```
@@ -28,29 +34,499 @@ for (const [key, value] of Object.entries(schemas)) {
+    } else if (currentSchema === "runCode_v2") {
+      transformedObject.runCode = {
+        language: object.language,
+        code: object.code,
+        args: object.args,
+        workingDirectory: object.workingDirectory,
+        exitCodes: object.exitCodes,
+        stdio: object.output,
+        path: object.savePath,
+        directory: object.saveDirectory,
+        maxVariation: object.maxVariation / 100,
+        overwrite: (object.overwrite = "byVariation"
```

Biome (1.9.4)

[error] Unexpected constant condition.



(lint/correctness/noConstantCondition)